code_schemes/idps_decision_making_opportunities_yes_no_amb.json

```diff
@@ -0,0 +1,157 @@
+{
+    "Version": "0.0.0.1",
+    "Name": "Idps Decision Making Opportunities Yes/No/Amb",
+    "SchemeID": "Scheme-d1d9cd35",
+    "Codes": [
+        {
+        "DisplayText": "Yes",
+        "VisibleInCoda": true,
+        "NumericValue": 0,
+        "CodeID": "code-0912d874",
+        "CodeType": "Normal",
+        "MatchValues": [
+          "yes"
+        ],
+        "StringValue": "yes"
+      },
+      {
+        "NumericValue": 1,
+        "CodeID": "code-378d2620",
+        "CodeType": "Normal",
+        "MatchValues": [
+          "no"
+        ],
+        "StringValue": "no",
+        "DisplayText": "No",
```

Comment:
Convention is usually for these to be lower case

Reply:
updated thanks